pyproject.toml

```
@@ -40,8 +44,18 @@ dev = [
   "isort>=5.13.2",
   "mypy>=1.9.0",
   "pre-commit>=4",
+  # Visualization (make sure these stay equalivalent to the 'visualizer' group)
+  "dash>=3.0.0",
+  "dash-bootstrap-components>=2.0.0",
+  "dash-cytoscape>=1.0.2",
+]
 
+visualizer = [
```

```suggestion
visualizer = [
  # Visualization (make sure these are also updated in the 'dev' group)
```
I expect people to first update the visualizer group if working with it, so reminding them in both places?